src/App.js

```diff
@@ -1,21 +1,21 @@
 import "./App.css";
 import { Routes, Route, Link } from "react-router-dom";
 import Navbar from "./components/Navbar";
-import Sidebar from "./components/Sidebar";
-import Searchbar from "./components/Searchbar";
-import NoteCard from "./components/NoteCard";
+import Signin from "./components/Signin";
+import HomePage from "./pages/HomePage";
+import Signup from "./components/Signup";
+import MockmanEs from "mockman-js";
 
 function App() {
   return (
     <div className="App">
       <Navbar />
-      <div className="app-container pt-1">
-        <Sidebar />
-        <Searchbar />
-        <NoteCard />
-      </div>
-
-      <Routes>{/* <Route path="/" element={< />} /> */}</Routes>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/signin" element={<Signin />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route path="/mock" element={<MockmanEs />} />
+      </Routes>
```

Comment:
CODE Suggestion:
Instead of adding routes in the App.js file, you could move it to a separate file say NotesAppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.